### DISCUSSION TITLE
Failed upgrade, migration error

### DISCUSSION THREAD

**Discussion by wannagofast83** [netbox-community/netbox#13766] [Help Wanted!]
We are trying to upgrade from 3.5.4 to 3.6.1 and getting an error when we run the upgrade script.

`nextbox_ui_plugin.SavedTopology.created_by: (fields.E300) Field defines a relation with model 'users.AdminUser', which is either not installed, or is abstract.
nextbox_ui_plugin.SavedTopology.created_by: (fields.E307) The field nextbox_ui_plugin.SavedTopology.created_by was declared with a lazy reference to 'users.adminuser', but app 'users' doesn't provide model 'adminuser'.`

**Comment by candlerb** [comment]
Looks like you have a broken plugin installed, whose name is "nextbox_ui_plugin" ??

Try commenting it out from PLUGINS in configuration.py, then upgrading.

**Reply by wannagofast83** [reply]
I commented it out and the migration completed and it is working. Do I manually update the plugin to get it updated?

**Reply by candlerb** [reply]
You'll need to check if the plugin supports Netbox 3.6, and/or report the problem to the plugin author.

**Comment by chrisrik** [comment]
any update on this? Same issue when trying to install the plugin on a fresh netbox docker-installation

**Reply by candlerb** [reply]
The best chance of getting a solution is to raise the problem with the plugin author via the plugin's github repo.
